lib/index.js

```diff
@@ -24,6 +24,10 @@
     if (diff.length)
       throw new OptionError('Missing required options: ' + diff.join());
 
+    //Make url an array for multiple webhooks
+    if(typeof opts.url === 'string')
```

Comment:
You are right, actually it was on top of my head to do this but i forgot haha
New commit, i think this is the simplest way, maybe